src/eko/anomalous_dimensions/aem1.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+import numba as nb
+
+from .. import constants
+from . import as1
```

Comment:
Can you make all the anomalous dimension files consistent with this new convention? 
I would say `qcd_lo` and similar is more clear.

Reply:
most likely we'll go back to `lo` - without prefix - since this also becomes ambiguous at `nlo` as there are three possibilities ... instead, maybe we go for `lo1, lo2`, `nlo1`, `nlo2`, `nlo3` as is usually done in the EW literature ...

Reply:
No wait, let's stick to powers, they are much more straightforward than `lo1234` and so on.

If it is ambiguous to have a default to apply when the power it's not specified, let's always specify the power:
- `as1a0`
- `as0a1`
- `as2a0`
- `as1a1`
- ...

Reply:
well that's what we agreed on in the meeting on Monday (and that's why we asked in the first place)

Reply:
Why don't you like the fully explicit thing?

`nlo1`, `nlo2`, `nlo3` is certainly far more confusing...

Reply:
yes I agree, let's specify the powers, but in all files!

Reply:
as said on Monday... we give up on `gamma_ns_0` something we don't like

Reply:
At least for the modules, let's stick to the powers. Concerning the name of the functions, we can discuss, but I guess there is no need for suffixes, since they are scoped in modules.